robot_systems.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Robot:
-    pass
+    elevator = subsystem.Elevator()
 
 
 class Pneumatics:
```

Comment:
@Inspirol I think we can get rid of this Pneumatics class... right?

Reply:
Probably, unless we really need it. For now, I think we can keep it here and remove it in the dev branch once everything has been pull requested